FrontEnd/src/global.css

```diff
@@ -0,0 +1,44 @@
+ @import url('https://fonts.googleapis.com/css2?family=Inter&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+:root {
+  /* background color*/
+  --main-header-color: #1F9A7C;
+  --main-footer-background: #3C4044;
+  --new-members-section-background: #F1FFF7;
+  --why-should-section-background: #F0F1F3;
+  --copyright-info-background: #333333;
+  --about-as-main-background: #0B6C61;
+  --star-background: #FFD800;
+  --hint-background: #25292C;
+
+  /* text color*/
+  --main-text-color: #292E32;
+  --header-bunner-text-color:#000000;
+  --light-text-color: #FFFFFF;
+  --light-button-text-color: #FFFFFF;
+  --delete-text-color: #F15831;
+  --notification-text-color:#F34444;
+  --footer-dark-text-color: #A0ABC0;
+
+  /* button color*/
+  --main-button-color:#1F9A7C;
+  --light-button-color:#FFFFFF;
+  
+  scroll-behavior: smooth;
+  font-family: 'Inter', sans-serif;
+}
+
+a {
+  text-decoration: none;
+}
+
+.img {
```

Comment:
Do you want to apply this style to all `img` tags or classes?

Reply:
It should be for all img, but I accidentally put a period. I also had to comment on this part because if you connect global.css, some imgs float.

Reply:
Do we need this style at all? I think we can remove it

Reply:
It is really better to delete it.